css/app.scss

```diff
@@ -0,0 +1,98 @@
+
+@import "variables";
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: $font-source;
+    font-weight: $font-weight-normal;
+    font-size: $font-size-small;
+    line-height: $line-height;
+}
+a {
+    text-decoration: none;
+}
+h1
+{
+     font-weight: $font-weight-bold;  
+     font-size: $font-size-title;
+}
+h2
+{
+    font-weight: $font-weight-bold;  
+    font-size: $font-size-medium;
+}
+h3
+{
+    font-weight: $font-weight-bold;    
+}
+.btn-primary{
+    color: $primary-color;
+    background-color: transparent;
+    padding: 0.5rem 1rem;
+    border: 2px solid $primary-color;
+    text-transform: uppercase;
+    font-size: $font-size-small;
+    font-weight: $font-weight-bold;
+    letter-spacing: .1rem;
+    display: inline-block;
+    border-radius: 4px;
+}
+.newteaser__main{
+    display: flex;
+    padding:0 30px;
+    .newteaser__column{
+        width: 31.33%;
+        padding-right:25px;
+
+        &:last-child{
+            padding-right:0;
+        }
+        .newteaser__img {
+            position: relative;
+          
+            img{
+                width:100%;
+                height: 300px;            
+                object-fit: cover;
+                @media screen and (max-width: $small) {
+                    height: auto;
+                }
+            }
+            &:after {
+                content:'';
+                display: block;
+                position: absolute;
+                bottom:0;
+                width: 100%;
+                height:10px;
+                background:$primary-color;
+                left:0;
+                right:0;
+
+            }
+           
+        }
+        .newteaser__content{
+            .newteaser__content--heading{
+                    margin-bottom: 10px;
+                    line-height: $line-height;
+            }
+            .newteaser__content--text{
+                margin-bottom: 10px;
+                margin-top:0;
+            }
+          
+        }
+        @media screen and (max-width: $small) {
```

Comment:
this is not the way to include media queries can u please refer moreor else I can help

Reply:
i have refer and update the changes